packages/langium/src/parser/langium-parser.ts

```diff
@@ -275,12 +275,6 @@ export class LangiumParser extends AbstractLangiumParser {
         };
     }
 
-    private appendHiddenTokens(tokens: IToken[]): void {
-        for (const token of tokens) {
-            this.nodeBuilder.addHiddenNode(token);
-        }
-    }
-
     private getHiddenTokens(token: IToken): IToken[] {
```

Comment:
GET is misleading, because while reading you will maybe assume that the structure stays untouched. I suggest POP or EXTRACT.